src/components/content-preview/content-preview.tsx

```diff
@@ -21,17 +97,34 @@ export default function ContentPreview({
           {title}
         </TypographyScale>
         <Grid columns={[1, null, 2, 3]} sx={styles.ContentPreviewGrid}>
-          {content.map(card => {
-            return (
-              <Card
-                key={card.title} // TODO: better keys when actual data used
-                tag={card.tag}
-                title={card.title}
-                count={card.count}
-                description={card.description}
-              />
-            );
-          })}
+          {content.map(
+            ({
+              id,
+              slug,
+              title,
+              lessons,
+              contentType,
+              externalLink,
+              fileDownload,
+              shortDescription,
+            }) => {
```

Comment:
Personal preference, but since we already have a `title` prop in this component, it might be good to be more explicit about this `title` variable that we are using here just to avoid confusion.

Reply:
agreed, saw this when i was doing my self-review. I'll update